public/app/features/explore/Logs/LogsVolumePanelList.tsx

```diff
@@ -97,8 +98,29 @@ export const LogsVolumePanelList = ({
   } else if (timeoutError) {
     return (
       <SupplementaryResultError
-        title="The logs volume query has timed out"
+        title="Unable to show log volume"
         // Using info to avoid users thinking that the actual query has failed.
+        message={
+          <>
+            <p>
+              <Trans i18nKey="explore.logs.logs-volume.much-data">
+                The query is trying to access too much data. Try one or more of the following:
+              </Trans>
+            </p>
+            <ul>
+              <li>
+                <Trans i18nKey="explore.logs.logs-volume.add-filters">
+                  Add more filters to your query to narrow down your search.
```

Comment:
We should be specific to add more "labels", as line filters wouldn't help with query performance.

```suggestion
                  Add more labels to your query to narrow down your search.
```